app/resonant-laboratory/general_purpose/binUtils.js

```diff
@@ -19,7 +19,11 @@ function coerceValue (value, coerceToType) {
   } else if (coerceToType === 'number') {
     value = parseFloat(value);
   } else if (coerceToType === 'string') {
-    value = String(value);
+    if (value && value.str) {
+      value = value.str;
```

Comment:
Not as far as `String()` is concerned... instead, we get "ObjectId(4a58dacf6d5fdc ... )" instead of JSON or the hash. This tweak just helps us avoid the "ObjectId(" decoration.

I guess `JSON.stringify()` might make more sense than `String()`? But if we try to coerce big, complicated objects, it's probably still best to just bin them under `[Object object]`...